lib/date.rb

```diff
@@ -0,0 +1,22 @@
+require 'date'
+
+require_relative 'frontdesk.rb'
+require_relative 'reservation.rb'
+
+module Hotel
+  class DateRange
+
+      def contains(date)
+        (date >= @start_date && date < @end_date) ? true : false
```

Comment:
Please use explicit an `return`; also, your condition is already a boolean so you can return it directly:

```suggestion
        return date >= @start_date && date < @end_date
```